docs/rules/prefer-map-has.md

```diff
@@ -0,0 +1,56 @@
+# Prefer `Map#has` over `Map#get`
+
+💼 This rule is enabled in the ✅ `recommended` [config](https://github.com/sindresorhus/eslint-plugin-unicorn#preset-configs-eslintconfigjs).
+
+💡 This rule is manually fixable by [editor suggestions](https://eslint.org/docs/latest/use/core-concepts#rule-suggestions).
+
+<!-- end auto-generated rule header -->
+<!-- Do not manually modify this header. Run: `npm run fix:eslint-docs` -->
+
+In some places where Boolean value judgment is required, it is more reasonable to prefer `Map#has` over `Math#get`.
```

Comment:
Needs a better description. Feel free to use ChatGPT to improve it.

Reply:
I have already been using it, but I haven't found a more suitable description. Could you help me with this?